waiter/src/waiter/middleware.clj

```diff
@@ -14,18 +14,18 @@
             [waiter.async-utils :as au]
             [waiter.utils :as utils]))
 
-(defn wrap-context
-  "Wraps a handler, ensuring the request and response (or exception) contains data in the context map."
-  [handler context]
+(defn wrap-update
```

Comment:
Should we have `assoc` and `merge` versions which ends up using `update` under the covers? That makes the user code simpler to read instead of having to create the lambdas everywhere.

Reply:
I thought about going down that path, but merge isn't used and assoc used only once.  I'll create `wrap-assoc`.